contrib/stagit/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "stagit"
+pkgver = "1.2"
+pkgrel = 0
+build_style = "makefile"
+make_build_args = [
+    "LIBGIT_INC=-I/usr/include",
+    "LIBGIT_LIB=-L/usr/lib -lgit2",
```

Comment:
```suggestion
    "LIBGIT_LIB=-lgit2",
```

default lib path already